Tests failing on Windows

### Current Behavior

Getting error:

```
reposgithub: Cannot find module 'C:Userssdmcl
Flowforge-dev-envpackages
Flowforge-nr-launcherlibauthadminAuth.js'
Require stack:
- C:\Users\sdmcl\AppData\Local\Temp\fflauncher-iuB3Vt\9a9z43diiiv.js
- C:\Users\sdmcl\repos\github\flowforge-dev-env\packages\flowforge-nr-launcher\test\unit\lib\runtimeSettings_spec.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:985:15)
      at Function.Module._load (node:internal/modules/cjs/loader:833:27)
      at Module.require (node:internal/modules/cjs/loader:1057:19)
      at require (node:internal/modules/cjs/helpers:103:18)
      at Object.<anonymous> (C:\Users\sdmcl\AppData\Local\Temp\fflauncher-iuB3Vt\9a9z43diiiv.js:6:16)
      at Module._compile (node:internal/modules/cjs/loader:1155:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1209:10)
      at Module.load (node:internal/modules/cjs/loader:1033:32)
      at Function.Module._load (node:internal/modules/cjs/loader:868:12)
      at Module.require (node:internal/modules/cjs/loader:1057:19)
      at require (node:internal/modules/cjs/helpers:103:18)
      at loadSettings (test\unit\lib\runtimeSettings_spec.js:30:16)
      at async Context.<anonymous> (test\unit\lib\runtimeSettings_spec.js:338:26)
```

### Expected Behavior

Tests should run

### Steps To Reproduce

`npm run test` on windows

### Environment

- FlowForge version:
- Node.js version:
- npm version:
- Platform/OS: Windows 11
- Browser:

--- a/test/unit/lib/runtimeSettings_spec.js
+++ b/test/unit/lib/runtimeSettings_spec.js
@@ -22,7 +22,7 @@ describe('Runtime Settings', function () {
         const nmPath2 = path.normalize(path.join(__dirname, '../../../../../node_modules')).split(path.sep).join('/')
         // Rewrite any requires for exports of this module
         content = content.replace(/'(@flowforge\/nr-launcher\/.*)'/g, (match, p1) => {
-            return `'${require.resolve(p1)}'`
+            return `'${require.resolve(p1).split(path.sep).join('/')}'`
         })
         content = `module.paths.unshift('${nmPath}', '${nmPath2}'); ${content}`
         const fn = path.normalize(path.join(TMPDIR, `${Math.random().toString(36).substring(2)}.js`)).split(path.sep).join('/')

